### PR TITLE
Minor fixes to slimer

### PR DESCRIPTION
**Pull request by jdextercsc**
Issues resolved:
- mtu size for bond and bonded vlan interfaces is now a variable that is set in either group_vars or by host.
- subscription-managers pool is using var "pool" which is a string containing the name of the pool that is to be used.  This was changed from the command module for idepotency.
- rabbitmq-server failed to start on subsuquent runs after the firewall role was run.  Added port 25672 to controllers to allow for the clustering of rabbit servers.
- enabled stonith for PCS.

Recaps below from test run.

2015-09-21 16:39:31,098 p=5953 u=root |  PLAY RECAP ********************************************************************
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-cmp-1                : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-ctlr-1               : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-ctlr-2               : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-ctlr-3               : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-kragle               : ok=4    changed=0    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-scale-1              : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-scale-2              : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-scale-3              : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-swift-1              : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-swift-2              : ok=15   changed=7    unreachable=0    failed=0
2015-09-21 16:39:31,099 p=5953 u=root |  wiley-swift-3              : ok=15   changed=7    unreachable=0    failed=0

2015-09-21 17:39:59,282 p=12234 u=root |  PLAY
2015-09-21 17:39:59,282 p=12234 u=root |  wiley-cmp-1                : ok=75   changed=54   unreachable=0    failed=0
2015-09-21 17:39:59,282 p=12234 u=root |  wiley-ctlr-1               : ok=344  changed=260  unreachable=0    failed=0
2015-09-21 17:39:59,282 p=12234 u=root |  wiley-ctlr-2               : ok=348  changed=265  unreachable=0    failed=0
2015-09-21 17:39:59,282 p=12234 u=root |  wiley-ctlr-3               : ok=348  changed=265  unreachable=0    failed=0
2015-09-21 17:39:59,283 p=12234 u=root |  wiley-scale-1              : ok=27   changed=23   unreachable=0    failed=0
2015-09-21 17:39:59,283 p=12234 u=root |  wiley-scale-2              : ok=27   changed=23   unreachable=0    failed=0
2015-09-21 17:39:59,283 p=12234 u=root |  wiley-scale-3              : ok=27   changed=23   unreachable=0    failed=0
2015-09-21 17:39:59,283 p=12234 u=root |  wiley-swift-1              : ok=80   changed=55   unreachable=0    failed=0
2015-09-21 17:39:59,283 p=12234 u=root |  wiley-swift-2              : ok=80   changed=55   unreachable=0    failed=0
2015-09-21 17:39:59,283 p=12234 u=root |  wiley-swift-3              : ok=80   changed=55   unreachable=0    failed=0

2015-09-22 08:32:11,224 p=8258 u=root |  PLAY RECAP ********************************************************************
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-cmp-1                : ok=10   changed=6    unreachable=0    failed=0
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-ctlr-1               : ok=29   changed=19   unreachable=0    failed=0
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-ctlr-2               : ok=29   changed=19   unreachable=0    failed=0
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-ctlr-3               : ok=29   changed=19   unreachable=0    failed=0
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-scale-1              : ok=28   changed=16   unreachable=0    failed=0
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-scale-2              : ok=28   changed=16   unreachable=0    failed=0
2015-09-22 08:32:11,224 p=8258 u=root |  wiley-scale-3              : ok=26   changed=14   unreachable=0    failed=0

### CODE REVIEW

**Comment by jcpowermac** [comment]
@jdextercsc can we get a `pcs status` listing, just to confirm that STONITH is working properly.

File: compute-nodes.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
         peerdns: '"no"'
         master: bond0
         slave: '"yes"'
-        mtu: 9000
+        mtu: "{{mtu | default (8964) }}"
```
Comment (jcpowermac) [inline compute-nodes.yml]: Should the default be something standard like 1500 or 9000 and not something that will just work in our environment?

**Comment by mtnbikenc** [comment]
The default MTU should not be set per task in the play.  Since this is a role, it should go in roles/iface/defaults/main.yml. I feel a default of 1500 would be more appropriate.  To set the MTU for the project, we would put the var under inventory/group_vars to override the role defaults (see Ansible Variable Precedence).

There are references to MTU in other places in the project (nova & neutron) and they are set to 9000.  These should be taken into consideration since MTU is being variablized.

**Comment by jdextercsc** [comment]
```[root@wiley-ctlr-1 ~]# pcs status
Cluster name: automation_cluster
Last updated: Tue Sep 22 11:06:34 2015      Last change: Tue Sep 22 10:55:45 2015 by root via cibadmin on wiley-ctlr-1
Stack: corosync
Current DC: wiley-ctlr-1 (version 1.1.13-a14efad) - partition with quorum
3 nodes and 124 resources configured

Online: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]

Full list of resources:

 ipmilan-fence-wiley-ctlr-1 (stonith:fence_ipmilan):    Started wiley-ctlr-2
 ipmilan-fence-wiley-ctlr-2 (stonith:fence_ipmilan):    Started wiley-ctlr-1
 ipmilan-fence-wiley-ctlr-3 (stonith:fence_ipmilan):    Started wiley-ctlr-1
 Clone Set: haproxy-clone [haproxy]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 vip-db (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-msg    (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 vip-keystone-int   (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-glance-int (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-1
 vip-cinder-int (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 vip-nova-int   (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-neutron-int    (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-1
 vip-horizon-int    (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 vip-heat-int   (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-ceilometer-int (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-1
 vip-swift-int  (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 vip-scaleio-int    (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-keystone-pub   (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-1
 vip-glance-pub (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 vip-cinder-pub (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-nova-pub   (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-1
 vip-neutron-pub    (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 vip-horizon-pub    (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-1
 vip-heat-pub   (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-3
 vip-ceilometer-pub (ocf::heartbeat:IPaddr2):   Started wiley-ctlr-2
 Clone Set: memcached-clone [memcached]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: rabbitmq-clone [rabbitmq]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Master/Slave Set: galera-master [galera]
     Masters: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: mongodb-clone [mongodb]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: keystone-clone [keystone]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: glance-registry-clone [glance-registry]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: glance-api-clone [glance-api]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: cinder-api-clone [cinder-api]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: cinder-scheduler-clone [cinder-scheduler]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 cinder-volume  (systemd:openstack-cinder-volume):  Started wiley-ctlr-3
 Clone Set: neutron-server-clone [neutron-server]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: neutron-scale-clone [neutron-scale](unique)
     neutron-scale:0    (ocf::neutron:NeutronScale):    Started wiley-ctlr-1
     neutron-scale:1    (ocf::neutron:NeutronScale):    Started wiley-ctlr-2
     neutron-scale:2    (ocf::neutron:NeutronScale):    Started wiley-ctlr-3
 Clone Set: neutron-ovs-cleanup-clone [neutron-ovs-cleanup]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: neutron-netns-cleanup-clone [neutron-netns-cleanup]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: neutron-openvswitch-agent-clone [neutron-openvswitch-agent]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: neutron-dhcp-agent-clone [neutron-dhcp-agent]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: neutron-l3-agent-clone [neutron-l3-agent]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: neutron-metadata-agent-clone [neutron-metadata-agent]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: nova-novncproxy-clone [nova-novncproxy]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: nova-api-clone [nova-api]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: nova-scheduler-clone [nova-scheduler]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: nova-conductor-clone [nova-conductor]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 nova-consoleauth   (systemd:openstack-nova-consoleauth):   Started wiley-ctlr-1
 Clone Set: heat-api-clone [heat-api]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: heat-api-cfn-clone [heat-api-cfn]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: heat-api-cloudwatch-clone [heat-api-cloudwatch]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 heat-engine    (systemd:openstack-heat-engine):    Started wiley-ctlr-2
 ceilometer-central (systemd:openstack-ceilometer-central): Started wiley-ctlr-3
 Clone Set: ceilometer-collector-clone [ceilometer-collector]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: ceilometer-api-clone [ceilometer-api]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: ceilometer-alarm-evaluator-clone [ceilometer-alarm-evaluator]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: ceilometer-alarm-notifier-clone [ceilometer-alarm-notifier]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: ceilometer-notification-clone [ceilometer-notification]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: ceilometer-delay-clone [ceilometer-delay]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 Clone Set: swift-proxy-clone [swift-proxy]
     Started: [ wiley-ctlr-1 wiley-ctlr-2 wiley-ctlr-3 ]
 swift-object-expirer   (systemd:openstack-swift-object-expirer):   Started wiley-ctlr-1

PCSD Status:
  wiley-ctlr-1: Online
  wiley-ctlr-2: Online
  wiley-ctlr-3: Online

Daemon Status:
  corosync: active/enabled
  pacemaker: active/enabled
  pcsd: active/enabled```

**Comment by jcpowermac** [comment]
Code review :+1:

File: roles/neutron/tasks/juno.yml [hunk 2]
```diff
@@ -136,7 +136,7 @@
     - { section: "DEFAULT", option: "verbose", value: "true"}
     - { section: "agent", option: "tunnel_types", value: "{{ neutron_agent_tunnel_types }}"}
     - { section: "agent", option: "vxlan_udp_port", value: "{{ neutron_vxlan_udp_port }}"}
-    - { section: "agent", option: "veth_mtu", value: "9000"}
+    - { section: "agent", option: "veth_mtu", value: "{{mtu | default 1500 }}"}
```
Comment (mtnbikenc) [inline roles/neutron/tasks/juno.yml]: Add parentheses to the default value.  Like was done for nova-compute.

**Comment by mtnbikenc** [comment]
A codebase search for '9000' turned up these, which should be addressed.

https://github.com/csc/slimer/blob/e49ad41c8de4185b6134988f59a32f15ab602cb2/inventory/group_vars/all/common.yml#L130
Var is set (and commented) but not used anywhere.  Can it be removed?

https://github.com/csc/slimer/blob/e49ad41c8de4185b6134988f59a32f15ab602cb2/roles/neutron/defaults/main.yml#L51
Var is set but not used anywhere.  Can it be removed?

https://github.com/csc/slimer/blob/e49ad41c8de4185b6134988f59a32f15ab602cb2/roles/neutron/tasks/kilo.yml#L141
Should use mtu var

https://github.com/csc/slimer/blob/e49ad41c8de4185b6134988f59a32f15ab602cb2/roles/nova-compute/tasks/kilo.yml#L147
Should use mtu var

**Comment by mtnbikenc** [comment]
So, I wasn't asking for all variables to be cleaned up.  I just wanted things that referenced MTU to be corrected or removed.  There is a larger variable clean-up task that needs to be undertaken.

**Comment by jdextercsc** [comment]
PLAY RECAP ********************************************************************
wiley-cmp-1                : ok=75   changed=54   unreachable=0    failed=0
wiley-ctlr-1               : ok=345  changed=261  unreachable=0    failed=0
wiley-ctlr-2               : ok=349  changed=266  unreachable=0    failed=0
wiley-ctlr-3               : ok=349  changed=266  unreachable=0    failed=0
wiley-scale-1              : ok=27   changed=23   unreachable=0    failed=0
wiley-scale-2              : ok=27   changed=23   unreachable=0    failed=0
wiley-scale-3              : ok=27   changed=23   unreachable=0    failed=0
wiley-swift-1              : ok=80   changed=55   unreachable=0    failed=0
wiley-swift-2              : ok=80   changed=55   unreachable=0    failed=0
wiley-swift-3              : ok=80   changed=55   unreachable=0    failed=0

**Comment by jcpowermac** [comment]
Code review :+1:

**Comment by mtnbikenc** [comment]
Looks good. Thanks for making the changes and retesting. :+1:

**Pull request merged**